### PR TITLE
Bugfix handling of environment tool handling + more checks + refactor.

### PR DESCRIPTION
**Pull request by vinjana**

### CODE REVIEW

File: resources/analysisTools/roddyTools/wrapInScript.sh [hunk 1]
```diff
@@ -48,20 +57,44 @@ declare_xg() {
     eval "export $varName=\"$value\""
 }
 
+getEnvironmentScriptPath() {
```
Comment (dankwart-de) [inline resources/analysisTools/roddyTools/wrapInScript.sh]: Some comments? An example?

File: resources/analysisTools/roddyTools/wrapInScript.sh [hunk 2]
```diff
@@ -48,20 +57,44 @@ declare_xg() {
...
+
 # Basic modules / environment support
 # Load the environment script (source), if it is defined. If the file is defined but the file not accessible exit with
 # code 200. Additionally, expose the used environment script path as ENVIRONMENT_SCRIPT variable to the wrapped script.
 runEnvironmentSetupScript() {
     local envScriptVar="${TOOL_ID}EnvironmentScript"
     if [[ -n "${!envScriptVar}" ]]; then
-        # declare -gx ENVIRONMENT_SCRIPT="${!envScriptVar}"
-        declare_xg ENVIRONMENT_SCRIPT "${!envScriptVar}"
+	    warnEnvironmentScriptOverride
```
Comment (dankwart-de) [inline resources/analysisTools/roddyTools/wrapInScript.sh]: That's basically the same code as for the else branch. You could reduce it to two method calls and implement a new method. But it is ok like it is.

**Review by dankwart-de — APPROVED** [review]
We have so many nice methods now. Maybe we shall move them all to one or more separate files.

File: resources/analysisTools/roddyTools/wrapInScript.sh (hunk 1 above)
Comment (vinjana) [reply resources/analysisTools/roddyTools/wrapInScript.sh]: Done.

File: resources/analysisTools/roddyTools/wrapInScript.sh (hunk 2 above)
Comment (vinjana) [reply resources/analysisTools/roddyTools/wrapInScript.sh]: I'll change it.

**Review by dankwart-de — APPROVED** [review]

**Pull request merged**